README.md

```diff
@@ -59,40 +59,48 @@ The leak tracker can catch only certain types of leaks, in particular, related t
 
 The tool assumes that, with proper memory management,
 an object's disposal and garbage collection should happen sequentially,
-close to each other.
+close to each other. I.e. the object should be garbage collected
+during next garbage collection cycle after disposal.
 
 By monitoring disposal and Garbage Collect events, the tool detects different types of leaks:
 
 - **Not disposed, but GCed (not-disposed)**:
-    - a disposable object was GCed,
+
+    - **Definition**: a disposable object was GCed,
        without being disposed first. This means that the object's disposable content
        is using memory after the object is no longer needed.
-       To fix the leak, invoke `dispose()` to free up the memory.
+
+    - **Fix**: invoke `dispose()` for the object to free up the memory.
 
 - **Disposed, but not GCed (not-GCed)**:
-    - an object was disposed,
+    - **Definition**: an object was disposed,
        but not GCed after certain number of GC events. This means that
        a reference to the object is preventing it from being
        garbage collected after it's no longer needed.
-       To fix the leak, after disposal assign all reachable references
-       of the object to null:
 
-    ```
-    myField.dispose();
-    myField = null;
-    ```
+    - **Fix**: To fix the leak, assign all reachable references
+       of the object to null after disposal:
+
+        ```
+        myField.dispose();
+        myField = null;
+        ```
 
 - **Disposed and GCed late (GCed-late)**:
-    - an object was disposed and then GCed,
+    - **Definition**: an object was disposed and then GCed,
        but GC happened later than expected. This means the retaining path was
        holding the object in memory for some period, but then disappeared.
 
+    - **Fix**: the same as for **not-GCed**
+
 - **Disposed, but not GCed, without path (not-GCed-without-path)**:
-    - an object
+    - **Definition**: an object
        was disposed and not GCed when expected, but retaining path is not detected,
        that means that the object will be most likely GCed in the next GC cycle,
        and the leak will convert to **GCed-late** leak.
 
+    - **Fix**: please, [create issue](https://github.com/dart-lang/leak_tracker/issues) if you see this type of leaks, as it means something is wrong with the tool.
```

Comment:
**issue:** This information would be better presented as a table. I would replace it with the following:
```suggestion
| Memory Leak Type          | Disposed | Garbage Collected | Explanation                                                                                                                                                                                                                                   | Fix                                                                                                                              |   |
|---------------------------|:--------:|:-----------------:|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|---|
| **not-disposed**          |    No    |        Yes        | The object's content continues to use memory after the program no longer needs the object.                                                                                                                                                    | Call `dispose()` to free up the memory.                                                                                          |   |
| **not-GCed**              |    Yes   |         No        | Memory disposed of an object, but did not free the memory that object occupied after certain number of GC events. A reference to the object prevents removing it from memory after the program no longer needs it to be in memory.            | Assign all reachable references of the object to `null` after disposal.  ```dart myField.dispose(); myField = null; ```          |   |
| **GCed-late**             |    Yes   |   Yes, but late   | Memory disposed of an object, then cleared the object from memory later than expected. The retaining path continued to hold the object in memory for some period, but then disappeared.                                                       | Assign all reachable references of the object to `null` after disposal.  ```dart myField.dispose(); myField = null; ```          |   |
| **not-GCed-without-path** |    Yes   |  No, without path | Memory disposed of an object and did not clear the object from memory when expected, but memory cannot detect the object's retaining path. The object should be cleared in the next GC cycle. The leak then converts to a **GCed-late** leak. | [Create an issue](https://github.com/dart-lang/leak_tracker/issues). If you see this type of leak, the tool might have an issue. |   |
```

Reply:
I love how the table look, but the table format makes it very hard to edit and review the page. So, leaving it bulleted list for the sake of maintainability.